cyences_app_for_splunk/default/data/ui/views/cs_ad_reports.xml

```diff
@@ -165,14 +165,14 @@
           <query>`cs_wineventlog_security` EventCode IN (4624, 4768, 4769) src=* [| search `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794) | dedup ComputerName Subject_Account_Name | table ComputerName Subject_Account_Name |  rename Subject_Account_Name as user] 
 | bin span=1m _time 
 | eval date_wday = strftime(_time, "%A"), date_hour = tonumber(strftime(_time, "%H")) 
-| `cs_ad_password_change_outside_working_hour_definition` 
+| `cs_password_change_outside_working_hour_definition` 
```

Comment:
@crmpandya  - We need to copy the old macro value into the new one before deleting the old macro.

On Upgrade steps, Copy definition of cs_ad_password_change_outside_working_hour_definition into cs_password_change_outside_working_hour_definition. Delete the old macro.

Reply:
Also, let's keep the macro name as **cs_outside_working_hour_definition**